docs/start.md

```diff
@@ -296,3 +296,22 @@ The remote policy service feature allows custom policy schemes beyond simple req
 The hook is called after the standard request type and operation checks. If the hook returns an error, Signatory denies signing the operation.
 
 See [the documentation](remote_policy.md) for more information
+
+--- 
+## Giving USB Port Access to Signatory in Docker Using Volume Mount
```

Comment:
this is called "USB passthrough" 

consider renaming this section to "Configuring USB passthrough for Signatory in Docker"

also, these instructions are likely to be host OS specific, it may be worth mentioning which OS and architecture they have been tested on.

Reply:
Yeah, it is better to leave it from signatory docs as it is docker-related knowledge any user can find elsewhere. I know I raised this issue, but I later realized it is unnecessary in our docs. I should have closed this issue once I realized it. My bad.

Reply:
removed